### PR TITLE
docs(topics): updated design

### LINKED ISSUE #3: Obsolete "Acid" references in docs
On the [design doc](https://github.com/Azure/brigade/blob/master/docs/topics/design.md) page, there are 11 different references to Acid (all in images). I'm pretty sure this is a previous name of the project?* I grepped a source checkout, & found no references to Acid.

* Ok, confirm, I can see some activity last week on this rename. #225.

### PR DESCRIPTION
**Pull request by technosophos**
Closes #3

### CODE REVIEW

File: docs/topics/design.md [hunk 1]
```diff
@@ -1,183 +1,236 @@
...
+Instead, it used a Redis queue for message passing and key/value storage. Other
+than some proof-of-concept work, the Lua engine never materialized. JavaScript's
+popularity made it a better choice.
+
+An original Kuberentes-oriented JavaScript engine was developed several months later.
+This was intended to be both a stand-alone component and a foundational piece for
+Brigade. Work was abandoned in favor of the Node.js worker pattern.
+
+In April of 2017, Brigade was designated as the third part of the Helm/Draft/Brigade
+ecosystem. At this point, it was renamed "Acid" (Acme Continious Integration & Deployment).
```
Comment (bacongobbler) [inline docs/topics/design.md]: Continuous*

File: docs/topics/design.md [hunk 2]
```diff
@@ -1,183 +1,236 @@
...
+
+Brigade reached a stability point in September 2017, and was re-renamed back from
+Acid to Brigade. Brigade was released publically under the MIT license in October
```
Comment (bacongobbler) [inline docs/topics/design.md]: publicly*

File: docs/topics/design.md [hunk 3]
```diff
@@ -1,183 +1,236 @@
...
+- Configuration, such as tokens, SSH keys, and other items necessary to wire up
+  a script run.
+- Secrets, which are opaque configuration items that are presumed to be non-public.
+  While configuration information is not guaranteed to be available inside of a
+  script, secrets are.
+- The VCS sidecar to use. (Default is the Git sidecar)
+
+One or more scripts may be executed within the context of a project. Brigade
+assumes that a default script will reside in the project's VCS repository at the
+relative path `./brigade.js`. Gateways may provide other ways of sendings
```
Comment (adamreese) [inline docs/topics/design.md]: `s/sendings/sending/`

File: docs/topics/design.md (hunk 1 above)
Comment (adamreese) [inline docs/topics/design.md]: `s/Continious/Continuous/`

File: docs/topics/design.md (hunk 2 above)
Comment (adamreese) [inline docs/topics/design.md]: `s/publically/publicly/`

**Comment by technosophos** [comment]
`<leader>ss` is a good thing. ☹️

**Comment by technosophos** [comment]
Updated.

**Comment by technosophos** [comment]
🐝 🐝 🐝

**Pull request merged**